src/Symfony/Component/Security/Core/Encoder/PlaintextPasswordEncoder.php

```diff
@@ -14,7 +14,7 @@
 use Symfony\Component\Security\Core\Exception\BadCredentialsException;
 
 /**
- * PlaintextPasswordEncoder does not do any encoding.
+ * PlaintextPasswordEncoder does not do any encoding but is useful in testing environments.
  *
```

Comment:
"As this encoder is not cryptographically secure, usage of it in production environments is not recommended"

Thoughts as an added description?

Reply:
I would go one steps further and reword it to "[...] usage of it in production environments is discouraged."